Dockerfile

```diff
@@ -33,6 +33,9 @@ ENTRYPOINT ["./entrypoint"]
 
 FROM base AS no-build
 
+COPY --from=build-image /usr/local/lib /usr/local/lib
+RUN ldconfig
+
```

Comment:
I'm not sure if this is the best place to put it. I'm thinking about we applying an optimization afterwards, which will be applicable only for `no-build` and `dynamic-link`. But as of now, it is technically possible be moved to `base` (and also be applied to `static-link`), which avoids duplicated configuration.

Maybe it is time to create another shared stage for `no-build` and `dynamic-link`(?) This has some drawbacks, such as being required to duplicate packages in `{BUILD,RUNTIME}_TAR_PACKAGES` once again (due to using `COPY --from=build-image` in this shared stage).

Reply:
Do you mean something like `dynamic-base`? I don't know if it's worth it.

Reply:
It could be. I don't know either if its worth either.

A relevant point I now see is that it might be interesting for `static-link` as well. Any system library we build from source will require to be available in the IOC image, even with `STATIC_BUILD=YES`.

Reply:
As we discussed in person, I moved it to `base`.